.pryrc

```diff
@@ -0,0 +1,134 @@
+def create name='test card', content='some content', type='basic'
+  if name.kind_of? Hash
+    Card.create! name
+  elsif content.kind_of? Hash
+    Card.create!(content.merge(:name=>name))
+  else
+    Card.create! :name=>name, :content=>content, :type=>type
+  end
+end
+
+def update name='test card', *args
+  card_attr = {}
+  if args.first.kind_of? String
+    card_attr[:content] = args.shift
+    card_attr.merge!(args.first)
+  else
+    card_attr = args.first
+  end
+  Card.fetch(name).update_attributes card_attr
+end
+
+def _array
+ @_array ||= (1..6).to_a
+end
+
+def _hash
+  @_hash ||= {hello: "world", free: "of charge"}
+end
+
+def _user
+  @_user ||= Card.fetch 'Joe User'
+end
+
+def fe(name = nil)
+  if name
+    @fe = Card.fetch name
+  else
+    @fe ||= Card.fetch "home"
+  end
+end
+
+def cr(name = nil, content = 'some content', type = 'basic')
+  if name
+    @cr = create name, content, type
+  else
+    @cr ||= create
+  end
+end
+
+def ab
+  Card::Auth.as_bot
+end
+
+Pry.config.editor = proc { |file, line| "mine #{file}:#{line}" }
+
+Pry.config.commands.alias_command "h", "hist -T 20", desc: "Last 20 commands"
+Pry.config.commands.alias_command "hg", "hist -T 20 -G", desc: "Up to 20 commands matching expression"
+Pry.config.commands.alias_command "hG", "hist -G", desc: "Commands matching expression ever used"
+Pry.config.commands.alias_command "hr", "hist -r", desc: "hist -r <command number> to run a command"
+Pry.config.commands.alias_command "clear", "break --delete-all", desc: "remove all break points"
+
+if defined?(PryByebug)
+  Pry.commands.alias_command 'c', 'continue'
+  Pry.commands.alias_command 's', 'step'
+  Pry.commands.alias_command 'n', 'next'
+  Pry.commands.alias_command 'f', 'finish'
+end
+
+# Hit Enter to repeat last command
+Pry::Commands.command /^$/, "repeat last command" do
+  _pry_.run_command Pry.history.to_a.last
+end
+
+
+def hputs text
+  text = Nokogiri::XML(text, &:noblanks).root.to_s
+  puts CodeRay.scan(text, :html).term
+end
```

Comment:
I totally forgot that I wrote that one. Good to know :smile: In situations where syntax highlighting would have been useful I couldn't remember the details of this nokogiri/coderay combo and I didn't want to spend the time to look it up.

Reply:
yeah, very cool.  I just learned that teenagers are using "v" to mean very in conversation.

So I can just say "v kühl" to mean very cool.  Tip of the hat to Veronika.